<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="main_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="main_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="navigation_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="status_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>140</width>
         <height>158</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="home_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>158</y>
         <width>140</width>
         <height>96</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="scan_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>253</y>
         <width>140</width>
         <height>96</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="manage_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>348</y>
         <width>140</width>
         <height>96</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="setting_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>443</y>
         <width>140</width>
         <height>96</height>
        </rect>
       </property>
      </widget>
      <zorder>status_widget</zorder>
      <zorder>setting_widget</zorder>
      <zorder>manage_widget</zorder>
      <zorder>scan_widget</zorder>
      <zorder>home_widget</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest3">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
  <action name="actionTest3_2">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
